index.js

```diff
@@ -39,6 +39,9 @@ function isProtectedRoute(req) {
  */
 function isAuthenticated(site) {
   return function (req, res, next) {
+    // eslint-disable-next-line no-constant-condition
+    if (process.env.PREVENT_AUTH || true) return res.redirect(`${getAuthUrl(site)}/login`);
```

Comment:
Could you add this here? https://github.com/clay/amphora-auth/blob/fe2025cb499a3cd49537406eb063d45533034b3a/constants.js#L4 and we can ensure the string env var is cast to a bool and default it to false.